### PR TITLE
Move to Github Actions

### PR DESCRIPTION
**Pull request by jdtw**
Travis has been turned down for some time, so this PR moves to Github Actions instead.

### CODE REVIEW

File: pkg/server/storage/mysql_test.go [hunk 1]
```diff
@@ -37,6 +37,7 @@ func TestMysql(t *testing.T) {
 		Type:     "mysql",
 		Schema:   "sharkey_test",
 		Username: "root",
+		Password: "root",
```
Comment (jdtw) [inline pkg/server/storage/mysql_test.go:40]: There is no good way to set up mysql with an empty root password, it seems, so we need the tests to use a password.

File: .github/workflows/tests.yml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+jobs:
+
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Set up MySQL
+      uses: mirromutth/mysql-action@v1.1
```
Comment (mbyczkowski) [inline .github/workflows/tests.yml:17]: do we know which MySQL version this action uses?

File: .github/workflows/tests.yml [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+      with:
+        mysql database: 'sharkey_test'
+        mysql root password: 'root'
+
+    - name: Set up Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: 1.17
```
Comment (mbyczkowski) [inline .github/workflows/tests.yml:25]: nice bump to latest Go! 🎉  (I believe we were still running on Go 1.13 in Travis CI 😭)

File: pkg/server/storage/mysql_test.go (hunk 1 above)
Comment (mbyczkowski) [reply pkg/server/storage/mysql_test.go:40]: seems fine, but I wonder if that breaks local testing?

**Review by mbyczkowski — APPROVED** [review]

File: pkg/server/storage/mysql_test.go (hunk 1 above)
Comment (jdtw) [reply pkg/server/storage/mysql_test.go:40]: By default, mysql has an empty password. To set it you can do:

```
ALTER USER 'root'@'localhost' IDENTIFIED BY 'root';
```

And then local tests work:

```
$ mysql -e 'CREATE DATABASE sharkey_test;' -uroot -proot
$ go test -v ./...
```

File: .github/workflows/tests.yml (hunk 2 above)
Comment (jdtw) [reply .github/workflows/tests.yml:17]: Latest by default. Is there a version you'd prefer?

**Pull request merged**